### ISSUE TITLE
Support for file names/types overrides

### ISSUE THREAD

**Issue by krzychukula** [editorconfig/editorconfig-vscode#60]
```ini
# editorconfig.org
root = true

# Common rules
[*]
indent_size = 4
indent_style = space
end_of_line = lf
charset = utf-8
tab_width = 4
trim_trailing_whitespace = true
insert_final_newline = true

# Matches the exact files either *.json, *.yml or .htaccess
[*.jade]
indent_style = space
indent_size = 2

[package.json]
indent_style = space
indent_size = 2
```

Jade and package.json still use 4 spaces but they should use 2 spaces by default.

**Comment by jedmao** [comment]
Can you create a tiny repo that reproduces the issue? I can clone it and try to reproduce. Are any other plugin features working for you? I haven't run into this one.

**Comment by krzychukula** [comment]
Does anything else work for me: yes
Based on my tweaking of default config it's probably fault of tab_width not being overloaded. 
Still the same config works fine in other editors ;)

test repo: https://github.com/krzychukula/vs-editorconfig-demo

jade file should indent with 2 spaces.

**Comment by jedmao** [comment]
Thanks. I was able to reproduce and I believe the issue is [right here](https://github.com/editorconfig/editorconfig-vscode/blob/master/src/Utils.ts#L22-L24). Looking into it.

**Comment by jedmao** [comment]
@krzychukula you'll find the fix published in v0.3.1. I've also added a test in #61 to ensure it works and that it isn't reintroduced. Let me know how that works out for you!
